src/replication.c

```diff
@@ -3487,6 +3521,48 @@ void waitCommand(client *c) {
     replicationRequestAckFromSlaves();
 }
 
+/* WAIT for N replicas and / or local master to acknowledge our latest
+ * write command got synced to the disk. */
+void waitaofCommand(client *c) {
+    mstime_t timeout;
+    long numreplicas, numlocal, ackreplicas, acklocal;
+
+    /* Argument parsing. */
+    if (getRangeLongFromObjectOrReply(c,c->argv[1],0,1,&numlocal,NULL) != C_OK)
+        return;
+    if (getLongFromObjectOrReply(c,c->argv[2],&numreplicas,NULL) != C_OK)
```

Comment:
```suggestion
    if (getRangeLongFromObjectOrReply(c,c->argv[2],0,LONG_MAX,&numreplicas,NULL) != C_OK)
```

Reply:
i thought about it and decided to skip it because it was also missing from the other WAIT command.
but ok, let's change both..

Reply:
looking at this again, i think we better keep WAIT and WAITAOF similar in that respect, and i don't wanna change WAIT